network.go

```diff
@@ -161,6 +163,18 @@ func initNetwork() error {
 
 	networkManager = nm
 
+	lldpService = lldp.NewLLDP(&lldp.LLDPOptions{
+		InterfaceName: NetIfName,
+		EnableRx:      nc.LLDPMode.String != "disabled",
```

Comment:
Shouldn't this care about the tx_only and rx_only settings https://github.com/jetkvm/kvm/pull/948/files#diff-461417865cac9c4c5ea6a683e5f9f810e9a3f6cff4c8da5cd726ec6896dde346R764